exporter/containerimage/writer.go

```diff
@@ -525,6 +719,33 @@ func normalizeLayersAndHistory(ctx context.Context, remote *solver.Remote, histo
 	return remote, history
 }
 
+func removeLayerAnnotations(desc *ocispecs.Descriptor, oci bool) {
+	if oci {
+		// oci supports annotations but don't export internal annotations
+		delete(desc.Annotations, "containerd.io/uncompressed")
+		delete(desc.Annotations, "buildkit/createdat")
+		for k := range desc.Annotations {
+			if strings.HasPrefix(k, "containerd.io/distribution.source.") {
+				delete(desc.Annotations, k)
+			}
+		}
+	} else {
+		desc.Annotations = nil
+	}
+}
+
+func createAttestationsHistory(ctx context.Context, layers []ocispecs.Descriptor, attestations []exporter.Attestation) (history []ocispecs.History) {
```

Comment:
I think this can be just skipped. We don't care about history array for these configs.